cmd/zoekt-sourcegraph-indexserver/main_test.go

```diff
@@ -124,6 +250,109 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+func createTestNormalShard(t *testing.T, indexDir string, r zoekt.Repository, numShards int, optFns ...func(options *build.Options)) []string {
```

Comment:
mostly copied this from the builder package, should we create a `testutil` package?

Reply:
yup, maybe it's time. Not blocking for this PR though.